### PR TITLE
fix crafting terminal crash

### PR DESCRIPTION
**Pull request by asdflj**
fix in crafting terminal use hot key to transfer current crafting terminal item to craft slot crash

### CODE REVIEW

File: src/main/java/net/p455w0rd/wirelesscraftingterminal/common/container/ContainerWirelessCraftingTerminal.java [hunk 1]
```diff
@@ -1317,6 +1317,8 @@ public ItemStack slotClick(int slot, int button, int flag, EntityPlayer player)
                     && getSlot(slot) != null
                     && getSlot(slot).getStack() == RandomUtils.getWirelessTerm(player.inventory)) {
                 return null;
+            }else if(player.inventory.currentItem == button && flag == 2){
```
Comment (Alexdoru) [inline src/main/java/net/p455w0rd/wirelesscraftingterminal/common/container/ContainerWirelessCraftingTerminal.java]: you need to format your code like this to pass the spotless check 
image

or in your IDE you can run the spotlessJavaApply task
image

**Comment by Alexdoru** [comment]
that doesn't fix the crash entirely by the way, since you can also open the terminal via a keybind, and not necessarilly by holding it in your hand

**Comment by eigenraven** [comment]
The proper way to fix it is to make the slot with the terminal a SlotLocked from the ae2 slots classes

**Comment by Dream-Master** [comment]
@asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end.
https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8

**Comment by asdflj** [comment]
> @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)

this not good solution pls do not merge

**Comment by asdflj** [comment]
> The proper way to fix it is to make the slot with the terminal a SlotLocked from the ae2 slots classes

ur right

**Comment by Dream-Master** [comment]
> > @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)
> 
> this not good solution pls do not merge

Spotless not change any code just the format. Update Buildscript does not harm or what you mean ?

**Comment by asdflj** [comment]
> > > @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)
> > 
> > 
> > this not good solution pls do not merge
> 
> Spotless not change any code just the format. Update Buildscript does not harm or what you mean ?

I mean,I push the code is not good solution to fix crafting terminal  crash, sorry, My english is not good

**Comment by asdflj** [comment]
should be fine for now

**Review by eigenraven — APPROVED** [review]

**Pull request merged**